docs/user-guide/promql.md

```diff
@@ -0,0 +1,131 @@
+# PromQL
+
+## Introduction
+
+PromQL is the query language for Prometheus. It is a powerful and flexible language that can be used to retrieve data for alerting, graphing, and analysis.
+
+GreptimeDB has reimplemented PromQL in Rust, here is the compatibility list.
+
+## Compatibility List
+
+### Overview
+
+- Basics
+  - [Literal](#literal)
+  - [Selector](#selector)
+  - Subquery (unsupport)
+- Operators
+  - [Binary Operators](#binary-operators)
+  - Vector Matching (unsupport)
```

Comment:
Is it needed to use one of `unsupport` or `unsupported`?

Reply:
Looks "unsupport" is a wrong word, I've changed them to unsupported